nextcord/state.py

```diff
@@ -1695,6 +1690,9 @@ def parse_guild_member_remove(self, data) -> None:
                 data["guild_id"],
             )
```

Comment:
Remove this else block or update the log message as the event is still dispatched when the guild hasn't been found (raw.guild is None)